index.js

```diff
@@ -32,6 +33,23 @@ class Applesign {
     this.config = config.fromOptions(options || {});
     this.events = new EventEmitter();
     this.nested = [];
+    this.tmpDir = this._makeTmpDir();
+  }
+
+  _makeTmpDir () {
+    const tmp = tmpdir();
```

Comment:
there are modules and nodejs apis to create a temporary directory in a safe way, no need to uuid and concat paths

Reply:
any suggestions?